### PR TITLE
Couple of changes and a set of questions in comments

### PR DESCRIPTION
**Pull request by madninja**

### CODE REVIEW

**Comment by Vagabond** [comment]
I addressed all the comments except the one about stored_module_state, which I didn't want to address.

File: rebar.config [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {cover_export_enabled, true}.
 {cover_enabled, true}.
 {deps, [
-        {rocksdb, ".*", {git, "https://gitlab.com/barrel-db/erlang-rocksdb.git", {branch, "master"}}}
+        rocksdb
```
Comment (madninja) [inline rebar.config]: Want to pin a version?

File: src/relcast.erl [hunk 2]
```diff
@@ -189,16 +189,27 @@ start(ActorID, ActorIDs, Module, Arguments, RelcastOptions) ->
...
         {ok, ModuleState0} ->
             ModuleState = case rocksdb:get(DB, ActiveCF, <<"stored_module_state">>, []) of
                               {ok, SerializedModuleState} ->
                                   OldModuleState = Module:deserialize(SerializedModuleState),
                                   {ok, RestoredModuleState} = Module:restore(OldModuleState, ModuleState0),
                                   RestoredModuleState;
+                              not_found when PrevCF /= undefined ->
+                                  %% there might be a chance we crashed between creating the next column family
+                                  %% and writing the module state into it, so we check for this:
+                                  case rocksdb:get(DB, PrevCF, <<"stored_module_state">>, []) of
```
Comment (madninja) [inline src/relcast.erl]: nice!

File: src/relcast.erl [hunk 3]
```diff
@@ -636,8 +634,9 @@ find_next_outbound_(ActorID, {ok, <<"o", _/binary>> = Key, <<1:2/integer, ActorI
 find_next_outbound_(ActorID, {ok, <<"o", _/binary>>, <<1:2/integer, _/bits>>}, Iter, State) ->
     %% unicast message for someone else
     find_next_outbound_(ActorID, cf_iterator_move(Iter, next), Iter, State);
-find_next_outbound_(ActorID, {ok, <<"o", _/binary>> = Key, <<0:2/integer, Tail/bits>>}, Iter, State = #state{bitfieldsize=BitfieldSize}) ->
+find_next_outbound_(ActorID, {ok, <<"o", _/binary>> = Key, <<Type:2/integer, Tail/bits>>}, Iter, State = #state{bitfieldsize=BitfieldSize}) when Type == 0; Type == 2 ->
     <<ActorMask:BitfieldSize/integer-unsigned-big, Value/binary>> = Tail,
+    %% There's a lot of duplication between this and the next find_next_outbound head. Any way to simplify/combine?
```
Comment (madninja) [inline src/relcast.erl]: Remove this line if you're done with it

**Pull request merged**